src/Hl7.Fhir.Base/ElementModel/PocoBuilderExtensions.cs

```diff
@@ -28,8 +30,12 @@ public static T ToPoco<T>(this ITypedElement element, ModelInspector inspector,
         public static Base ToPoco(this ITypedElement element, ModelInspector inspector, PocoBuilderSettings settings = null) =>
                new NewPocoBuilder(inspector, settings ?? new PocoBuilderSettings()).BuildFrom(element);
 
-        public static ISourceNode ToSourceNode(this Base @base, ModelInspector inspector, string rootName = null) =>
-                @base.ToTypedElement(inspector, rootName).ToSourceNode();
-
+        public static ISourceNode ToSourceNode(this Base @base, ModelInspector inspector, string rootName = null)
```

Comment:
This file is called PocoBuilderExtensions, but now has other stuff too. This is no problem, but I think a re-org of the extension files is in order.

Reply:
See https://github.com/FirelyTeam/firely-net-sdk/issues/2992